### ISSUE TITLE
Thanos confusing docs

### ISSUE THREAD

**Issue by yasharne** [thanos-io/thanos#4319]
I'm trying to configure each Thanos components using the docs [here](https://thanos.io/tip/components/).
For example in the `query` component, states:

> If partial response strategy is NOT abort, this will “ignore” slower ...

But where should I define `partial response strategy`? There is no information about it in `thanos query --help` and in the [Partial Response Strategy](https://thanos.io/tip/components/query.md/#partial-response-strategy) section I cannot find any command line flag to define the strategy

**Comment by ianbillett** [comment]
@yasharne you can find information about the partial response strategy here: [docs/components/query.md#partial-response](https://github.com/thanos-io/thanos/blob/main/docs/components/query.md#partial-response).

TL;DR - it is a request parameter that defaults to 'abort'.

**Comment by yasharne** [comment]
Hi @ianbillett Thanks for your reply, but I still cannot see where I can define the strategy, the docs only mention there are some strategies but don't say anything about how to choose them

**Comment by yeya24** [comment]
@yasharne It is part of the query API. There are several ways to configure this behavior:
1. `query.partial-response` flag. You can configure this flag in querier to enable partial response, which means WARN (as opposed to ABORT, which means disallowing partial response)
2. You can set it in Thanos UI. Clicked means enabling partial response (WARN), otherwise it is ABORT.
![image](https://user-images.githubusercontent.com/25150124/121296475-9c28a300-c8a5-11eb-8650-dd55be60dc9a.png)

I will close this issue and feel free to reopen it in case you have more questions.

**Comment by ianbillett** [comment]
Super comment @yeya24 - I learnt something new today :raised_hands:
